### ISSUE TITLE
My Download has gone above 100 percent, 121.98%,  and is still going up and downloading.

### ISSUE THREAD

**Issue by Keyvantajfar** [bodaay/HuggingFaceModelDownloader#19]
My Download has gone above 100 percent, 121.98%,  and is still going up and downloading. 
I dunno what to do. the LFS temp folder that has 5 downloads i check their size and it is bigger than my target model.
about 21% higher as the download says. when will it stop and finishes the download? it just keeps going forever. it's been 2 nights now with my internet speed.

**Comment by Keyvantajfar** [comment]
> My Download has gone above 100 percent, 121.98%, and is still going up and downloading. I dunno what to do. the LFS temp folder that has 5 downloads i check their size and it is bigger than my target model. about 21% higher as the download says. when will it stop and finishes the download? it just keeps going forever. it's been 2 nights now with my internet speed.

i Also have a 9% next to the 120% idunno what is that, here's my terminal:
$ Downloading Phid-CodeLLaMA/TheBloke_Phind-CodeLlama-34B-v2-GGUF/phind-codellama-34b-v2.Q4_K_M.gguf Speed: 38.68 MB/sec, 123.37%    9%

**Comment by bodaay** [comment]
can you share the command and flags you passed, did you apply any filter?

**Comment by Keyvantajfar** [comment]
yes i'll see my command in my command history, i used at command so it gets
done during night. anyway. how i fixed it? i used "dd" and reduced those
incomplete .tmp files that were going more than 5 gb (they should have
stopped) and when i resumed the download started from 99% and got done.
first time i didnt use any flag and it got deleted after it couldnt
verify the checksum. but later i had saved the files, recovered and skipped
the checksum with your flag and it got ok. i still dont know if the file is
downloaded correctly or not.
thanks for your answer.

**Comment by yiweishe** [comment]
have same problem

Downloading models/RickyDeSkywalker_TheoremLlama/model-00001-of-00004.safetensors Speed: 99.08 MB/sec, 1447.32%     

When I go to the folder, there are 10x copies of the first 00001-of-00004.safetensors file.
